### ISSUE TITLE
QUESTION, CANNOT TRADE TOKEN???

### ISSUE THREAD

**Issue by ns7github** [forkdelta/tokenbase#2441]
Hello,

I was trying to trade a new token with it's contract address and symbol and the orders went to the orderbook perfectly. However, when I filled the order it shot blanks. No ETH was deposited when I did my buy order. The same for the other way around. Nothing was actually fulfilled regardless of me completing orders from the orderbook. It just says I made trades for 0 ETH. Everything says 0 ETH. I was trying to trade a brand new token without any special info on etherscan. Why does it shoot blanks and how do I fix this? There is like no purpose to the trade...

**Comment by freeatnet** [comment]
@ns7github can you please post torn token details and any transaction hashes you have for your transactions?

**Comment by ns7github** [comment]
Yes great here are the DEX trades on etherscan, this is the only record I have of anything: https://etherscan.io/token/0xe1a0a311adb89920816b79029484ea9b7d1ef32c#tokenTrade I traded on ForkDelta in the same series of trades and it all got labelled "Ether Delta" on Etherscan. You see NS7 is always 0. I was able to trade BAT token perfectly with no problems which is why I also opted to list NS7 wondering if its newness had something to do with this problem. In any event, my token right now shoots blanks lol! Hopefully that changes, thanks for the assistance.
